setup.py

```diff
@@ -46,11 +46,29 @@
         "psutil",
         "gunicorn==19.9.0",
         "six>=1.14.0",
+        "sphinx",
+        "sphinx_rtd_theme",
+        "sphinxcontrib-openapi",
+        "sphinxcontrib-redoc", 
         "boto3",
         "slack"
     ],
     extras_require={
-        "dev": ["tox", "pytest", "ipdb", "sphinx","sphinx_rtd_theme"]
+        "dev": [
```

Comment:
This may be a case where @michaelwoodruffdev 's older version is reinserting stuff @ccarterlandis removed. May require edit.